bom-2.346.x/pom.xml

```diff
@@ -47,6 +47,11 @@
                 <artifactId>ssh-slaves</artifactId>
                 <version>1.821.vd834f8a_c390e</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>trilead-api</artifactId>
+                <version>1.67.vc3938a_35172f</version>
```

Comment:
> this should be fixed in the trilead-api plugin

I expect it will be, but in the meantime trunk of `bom` is incorrect and there is a risk it could be released.

Reply:
Ah it is already banned: https://github.com/jenkins-infra/update-center2/pull/629

Reply:
Only tools going directly through Maven will see the newer release, update sites based ones won't.